app/build.gradle

```diff
@@ -198,6 +198,7 @@ dependencies {
     implementation project(":network")
     implementation project(":engelsystem")
 
+    api "androidx.lifecycle:lifecycle-viewmodel-ktx:2.5.1" // Added to Resolve Dependency Error -> https://issuetracker.google.com/issues/238425626#comment11
```

Comment:
Thank you for looking this up. I however do not see a reason to raise the version of `appCompat` and `coreKtx` in this commit. We can handle the dependency resolution conflict in the future. It might even be resolved by new versions of the androidx libraries. Please add a note to `appCompat` such as `Potential lifecycle-viewmodel-ktx conflict, see:  https://issuetracker.google.com/issues/238425626`.

Reply:
:heavy_check_mark: Thx for applying this.